src/test/java/org/mockitousage/junitrunner/StrictRunnerTest.java

```diff
@@ -55,6 +58,17 @@
         JUnitResultAssert.assertThat(result).fails(1, MyAssertionError.class);
     }
 
+    @Test public void runner_can_coexists_with_rule() {
```

Comment:
I have a different opinion on this. I would like to see mockito fail here. It indicates that the dev made something unintended and it is easy to fix.